intelmq/bots/collectors/xmpp/collector.py

```diff
@@ -21,32 +40,77 @@ def init(self):
             self.stop()
 
     def process(self):
-
         if self.xmpp is None:
-            self.xmpp = sleekxmpp.XMPPBot(self.parameters.xmpp_user + '@' + self.parameters.xmpp_server,
-                                self.parameters.xmpp_password,
-                                self.send_message,
-                                self.logger)
+            self.xmpp = XMPPClientBot(self.parameters.xmpp_user + '@'
+                                      + self.parameters.xmpp_server,
+                                      self.parameters.xmpp_password,
+                                      self.parameters.xmpp_room,
+                                      self.parameters.xmpp_room_nick,
+                                      self.parameters.xmpp_room_password,
+                                      self.logger)
             self.xmpp.connect(reattempt=True)
             self.xmpp.process()
 
+            # Add Handlers and register Plugins
+            self.xmpp.register_plugin('xep_0030')  # Service Discovery
+            self.xmpp.register_plugin('xep_0045')  # Multi-User Chat
+            self.xmpp.add_event_handler("message", self.log_message)
+            self.xmpp.add_event_handler("groupchat_message", self.log_message)
+
     def killbot(self):
         self.xmpp.disconnect(wait=True)
         self.logger.info("Disconnected")
 
+    def log_message(self, msg):
+        self.logger.debug("XMPP Received Event: %r , from %r", msg['body'],
```

Comment:
Stanza is a valid suggestion, as this term is widely used with xmpp. Will fix that.